app/grandchallenge/challenges/emails.py

```diff
@@ -26,20 +25,21 @@ def send_challenge_requested_email_to_reviewers(challengerequest):
         site_name=site.name,
         site_domain=site.domain,
     )
-    reviewers = get_user_model().objects.filter(
-        Q(groups__permissions__codename="change_challengerequest")
-        | Q(user_permissions__codename="change_challengerequest")
+    reviewers, _ = Group.objects.get_or_create(
```

Comment:
Should only be a get here.

Reply:
See comment above, shouldn't we better check for global permissions here as well for consistency, rather than group membership?

Reply:
Ah I think the look-up is more efficient this way actually.